content/docs/how-to-source-git.md

```diff
@@ -182,6 +182,17 @@ bcc2c8a292 resolved: create /etc/resolv.conf symlink at runtime
 1d39b39df9 Revert "journald: periodically drop cache for all dead PIDs"
 ```
 
+In case the patches are not in git format you need to apply them with `patch`:
+```
+$ patch -p1 --verbose --fuzz=0 <fedora/some-fix.patch
```

Comment:
What about removing those patches from the specfile?

Reply:
Not sure I understand. From downstream or upstream specfile? Why?

Reply:
Probably from both:
- in the source-git, the changes should in a form of commits
- in the dist-git, the changes should be in the form of auto-generated patches